regression-test/data/query_p0/sql_functions/array_functions/test_array_functions.out

```diff
@@ -704,15 +704,15 @@
 9	\N
 
 -- !select_array_shuffle1 --
-1	[1, 2, 3]	6	6	[3, 2, 1]	[3, 2, 1]
+1	[1, 2, 3]	6	6	[1, 3, 2]	[1, 3, 2]
```

Comment:
why these result changed ? did ur code will change other function ?

Reply:
<img width="574" alt="image" src="https://github.com/apache/doris/assets/25320945/f81eb18a-649b-423c-b2fb-0a85c659157a"

Seeding introduces randomness into the codes.

Reply:
is that determinate result ? if not, do not change other results , otherwise it can broke our pipeline tests